treeherder/seta/models.py

```diff
@@ -0,0 +1,44 @@
+from django.db import models
+from django.utils import timezone
+
+
+class TaskRequests(models.Model):
```

Comment:
Model names should be singular, not plural (i.e. `TaskRequest` not `TaskRequests`)